File: core/src/org/rexellentgames/dungeon/debug/LevelCommand.java
package org.rexellentgames.dungeon.debug;

import org.rexellentgames.dungeon.Dungeon;
import org.rexellentgames.dungeon.UiLog;

public class LevelCommand extends ConsoleCommand {
	{
		name = "/level";
		shortName = "/lvl";
		description = "Switches to given depth";
	}

	@Override
	public void run(Console console, String[] args) {
		if (args.length == 1) {
			Dungeon.goToLevel(Integer.valueOf(args[0]));
		} else {
			UiLog.instance.print("/level [id]");
		}
	}
}

File: core/src/org/rexellentgames/dungeon/debug/LevelCommand.kt
package org.rexellentgames.dungeon.debug

import org.rexellentgames.dungeon.Dungeon

class LevelCommand : ConsoleCommand("/level", "/lvl", "Changes level") {
    override fun run(console: Console, args: Array<String>) {
        if (args.size == 1) {
            Dungeon.goToLevel(Integer.valueOf(args[0]))
        }
    }
}